src/test/kotlin/no/nav/familie/ef/søknad/api/SøknadSkolepengerControllerTest.kt

```diff
@@ -45,58 +42,59 @@ class SøknadSkolepengerControllerTestConfiguration {
     fun featureToggleService(): FeatureToggleService = mockk()
 }
 
-@ActiveProfiles("local", "overgangsstonad-controller-test")
-@ExtendWith(SpringExtension::class)
-@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT, classes = [ApplicationLocalLauncher::class])
-internal class SøknadSkolepengerControllerTest {
+@ActiveProfiles("overgangsstonad-controller-test")
+internal class SøknadSkolepengerControllerTest : OppslagSpringRunnerTest() {
 
-    private fun serializedJWTToken() = JwtTokenGenerator.createSignedJWT(tokenSubject).serialize()
     @Autowired lateinit var søknadService: SøknadService
     @Autowired lateinit var featureToggleService: FeatureToggleService
 
-    @Value("\${local.server.port}")
-    val port: Int = 0
-    val contextPath = "/api"
     val tokenSubject = "12345678911"
 
+    @BeforeEach
+    fun førAlle() {
+        headers.setBearerAuth(søkerBearerToken(tokenSubject))
+    }
+
     fun søknadSkolepenger() = objectMapper.readValue(
-        File("src/test/resources/skolepenger/skolepenger.json"),
-        SøknadSkolepengerDto::class.java
+            File("src/test/resources/skolepenger/skolepenger.json"),
```

Comment:
Anbefaler å ta i bruk ktlint i prosjektet her å få riktige regler.
https://nav-it.slack.com/archives/G012YEK9YQ1/p1653998065811529